pkg/kubelet/apis/deviceplugin/v1beta1/constants.go

```diff
@@ -30,4 +30,8 @@ const (
 	DevicePluginPath = "/var/lib/kubelet/device-plugins/"
 	// KubeletSocket is the path of the Kubelet registry socket
 	KubeletSocket = DevicePluginPath + "kubelet.sock"
+	// Timeout duration in secs for PreStart RPC
+	KubeletPreStartRPCTimeoutInSecs = 30
```

Comment:
Reducing timeouts will be close to impossible in the future. Are we confident that `30 seconds` is necessary?
From a UX standpoint, the user will not know why the container is "stuck".
@vikaschoudhary16 @jiayingz

Reply:
@RenaudWasTaken based on our Wed's discussions in rmwg meeting, could you comment on whether 30s timeout is still necessary or we can make it shorter?